LayoutTests/transitions/resources/transition-test-helpers.js

```diff
@@ -261,6 +274,18 @@ function checkExpectedValue(expected, index)
         var expectedValues = parseClipPath(expectedValue);
         var values = parseClipPath(computedValue);
         
+        pass = false;
+        if (values && values.length == expectedValues.length) {
+            pass = true
+            for (var i = 0; i < values.length; ++i)
+                pass &= isCloseEnough(values[i], expectedValues[i], tolerance);
+        }
+    } else if (/(?:-webkit-)?border-(?:top|bottom)-(?:left|right)-radius$/.test(property)) {
+        computedValue = window.getComputedStyle(document.getElementById(elementId)).getPropertyCSSValue(property).cssText;
```

Comment:
Don’t think we need `window.` here.

Reply:
That's right, but this file is using `window.getComputedStyle` all over the place, so it seems better for consistency